### ISSUE TITLE
Feature request | Create Text area for X axis and Y axis Label

### ISSUE THREAD

**Issue by Vishalckc** [michaeldmoore/michaeldmoore-multistat-panel#47]
Currently I am unable to find any option for describing my x axis or Y axis labels. Will be a great help if I am able to do so
![image](https://user-images.githubusercontent.com/17704966/83903674-79295780-a756-11ea-8740-ce3c17b53b19.png)

**Comment by michaeldmoore** [comment]
Understood.  Best you can do for now is to write an explanatory panel title
- like "Number of widgets in thousands over time"

I may get to add something like this, but it's not going to be high on my
priority list.
Sorry

**Comment by Alexadra** [comment]
Hi.
@michaeldmoore Sorry for offtopic, but is there annotation support in your plugin?
Thanks

**Comment by michaeldmoore** [comment]
No.  I've never looked at the annotation API.  Is this something worth
looking into for a bar chart display?  What would it look like?

I might have a label option in the next version too - in the form of an
optional legend area.  Should be ready in a few weeks time.

**Comment by Alexadra** [comment]
It may be helpful in case when you want additionally to mark some event on a chart (e.g. deployment of new code version, software/hardware update, etc)
It looks like this on graph http://prntscr.com/t199n6 and may look similar on a bar chart (as a triangle on the bottom with tooltip on hover)
Currently, in Grafana you can add an Annotation from db http://prntscr.com/t19c8f
or by Ctrl+click on graph http://prntscr.com/t19bwh
Do you have an image how would labels look like?
